sdf.md

```diff
@@ -168,13 +169,10 @@ entity:
 ## Terminology and Conventions
 
 Thing:
-: A physical item that is also made available in the Internet of
-  Things.  The term is used here for Things that are notable for their
-  interaction with the physical world beyond interaction with humans;
-  a temperature sensor or a light might be a Thing, but a router that
-  employs both temperature sensors and indicator lights might exhibit
-  less Thingness, as the effects of its functioning are mostly on the
-  digital side.
+: A physical item that is also available over a network.
```

Comment:
```suggestion
: A physical item that is also available for interaction over a network.
```

Although maybe we don't need this term now? Or should we say that "Thing" is here simply a shorthand for SDF Thing?

Reply:
Object and sdfObject are the same, Thing and sdfThing are completely different.